services/core-web/src/index.js

```diff
@@ -19,12 +19,14 @@ import FeatureFlagProvider from "@mds/common/providers/featureFlags/featureFlag.
 const idleTimeout = 5 * 60_000;
 const refreshTokenBufferSeconds = 60;
 
-export const Index = () => {
+export const Index = (props) => {
   const [environment, setEnvironment] = useState(false);
 
-  fetchEnv().then(() => {
-    setEnvironment(true);
-  });
+  if (!props.disableEnvLoading) {
```

Comment:
Sonarcloud says disableEnvLoading missing in props validation, if we can add that somehow